draft-ietf-scitt-architecture.md

```diff
@@ -718,7 +718,7 @@ For instance, the code for the Registration Policy evaluation and endorsement ma
 The Transparency Service may be replicated with a consensus algorithm, such as Practical Byzantine Fault Tolerance {{PBFT}} and may be used to protect against malicious or vulnerable replicas.
 Threshold signatures may be use to protect the service key, etc.
 
-Issuers and Transparency Services MUST rotate verification keys for signature checking in well-defined cryptoperiods (see {{KEY-MANAGEMENT}}).
+Issuers and Transparency Services MUST rotate their keys, see {{KEY-MANAGEMENT}}.
```

Comment:
NIST Special Publication 800-57 Part 3
Cryptoperiod is defined in 

Revision 1
Recommendation for
Key Management
Part 3: Application-Specific Key 
Management Guidance